lib/dotcom_web/templates/page/_photo.html.heex

```diff
@@ -0,0 +1 @@
+<img src={"#{ @photo.image.url }"} class="h-120 w-full object-cover max-sm:hidden -z-10 relative" />
```

Comment:
suggestion: If we're not using alt text here, we can add `alt=""` to indicate to screen readers that this image is decorative.